Source/WebKit/UIProcess/API/glib/WebKitWindowProperties.cpp

```diff
@@ -395,13 +395,20 @@ void webkitWindowPropertiesUpdateFromWebWindowFeatures(WebKitWindowProperties* w
     webkitWindowPropertiesSetGeometry(windowProperties, &geometry);
 #endif
 
-    webkitWindowPropertiesSetMenubarVisible(windowProperties, windowFeatures.menuBarVisible);
-    webkitWindowPropertiesSetStatusbarVisible(windowProperties, windowFeatures.statusBarVisible);
-    webkitWindowPropertiesSetToolbarVisible(windowProperties, windowFeatures.toolBarVisible);
-    webkitWindowPropertiesSetLocationbarVisible(windowProperties, windowFeatures.locationBarVisible);
-    webkitWindowPropertiesSetScrollbarsVisible(windowProperties, windowFeatures.scrollbarsVisible);
-    webkitWindowPropertiesSetResizable(windowProperties, windowFeatures.resizable);
-    webkitWindowPropertiesSetFullscreen(windowProperties, windowFeatures.fullscreen);
+    if (windowFeatures.menuBarVisible)
+        webkitWindowPropertiesSetMenubarVisible(windowProperties, *windowFeatures.menuBarVisible);
+    if (windowFeatures.statusBarVisible)
+        webkitWindowPropertiesSetStatusbarVisible(windowProperties, *windowFeatures.statusBarVisible);
+    if (windowFeatures.toolBarVisible)
+        webkitWindowPropertiesSetToolbarVisible(windowProperties, *windowFeatures.toolBarVisible);
+    if (windowFeatures.locationBarVisible)
+        webkitWindowPropertiesSetLocationbarVisible(windowProperties, *windowFeatures.locationBarVisible);
```

Comment:
Nit: I know this is just making the existing code work with optionals.. but the discrepancy between Locationbar and locationBar, and Toolbar vs toolBar is confusing to me.

Reply:
I'm not going to change this as part of this commit, there are too many places where these names are like that on the WebKit API side.